### PR TITLE
Feature/initial content

### PR DESCRIPTION
**Pull request by nablanco**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,10 +18,13 @@
   },
   "dependencies": {
     "babel-plugin-styled-components": "^2.0.7",
+    "bootstrap": "^5.1.3",
```
Comment (ThomasEmilsson) [inline package.json]: Are you using this?

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Navbar, Container, Nav } from "react-bootstrap";
+
+const Header = () => {
```
Comment (ThomasEmilsson) [inline src/components/Header/Header.js:3]: Remove the bootstrap code if you're not planning on using it

File: src/pages/index.js [hunk 3]
```diff
@@ -1,184 +1,35 @@
...
+      <meta name="apple-mobile-web-app-capable" content="yes" />
+      <div id="home" className="content">
+        <title>Nicholas Blanco</title>
+        {/* <Button variant="primary">Primary</Button> */}
+        {/* <Header /> */}
+        <Hero />
+        <Resume />
+        {/* <About /> */}
+      </div>
+      {/* <Footer /> */}
```
Comment (ThomasEmilsson) [inline src/pages/index.js]: Remove commented code

**Review by ThomasEmilsson — APPROVED** [review]
Looks good, just minor comments about bootstrap and commented code. Consider updating later.

File: package.json (hunk 1 above)
Comment (nablanco) [reply package.json]: Forgot to remove! Thanks

File: src/components/Header/Header.js (hunk 2 above)
Comment (nablanco) [reply src/components/Header/Header.js:3]: Ya I'll go back in and clean up all the Bootstrap artifacts.. sorry!

**Pull request merged**